src/fundus/publishers/de/faz.py

```diff
@@ -50,3 +55,42 @@ def authors(self) -> List[str]:
         @attribute
         def title(self) -> Optional[str]:
             return self.precomputed.meta.get("og:title")
+
+    class V2(BaseParser):
+        _summary_selector = CSSSelector("div.header-teaser")
+        _paragraph_selector = CSSSelector("div.body-elements > p")
+        _subheadline_selector = CSSSelector("div.body-elements > h3")
+
+        _author_meta_selector = CSSSelector("div.author-meta")
+        _topic_selector = XPath("//div[text()=' Schlagworte: '] /a")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                summary_selector=self._summary_selector,
+                subheadline_selector=self._subheadline_selector,
+                paragraph_selector=self._paragraph_selector,
+            )
+
+        @attribute
+        def topics(self) -> List[str]:
+            topic_string = strip_nodes_to_text(self._topic_selector(self.precomputed.doc), join_on=",")
+            if topic_string is not None:
+                topic_string = topic_string.replace(",Alle Themen", "")
+                return generic_topic_parsing(topic_string, delimiter=",")
+            return []
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def authors(self) -> List[str]:
+            if self._author_meta_selector(self.precomputed.doc):
+                return generic_author_parsing(self.precomputed.ld.bf_search("author"))
+            return []
+
+        @attribute
+        def title(self) -> Optional[str]:
```

Comment:
I added a generic function to get titles from the root element. I don't mind leaving the URLs. Users have always the option to filter articles based on attributes.

Reply:
Imo, that option should just be used as a back-up since for most (maybe all) actual news articles the `og:title` element does exist and corresponds to the actual headline of the article, while the `<title>` element and headline usually do not seem to align.

Reply:
Hmm, i thought `<title>` would align better with the actual title. Good to know 👍

Reply:
Your reply came faster, than expexted. I had a little type in the commit suggestion, but you had already comitted it, before I edited it.